gem/static/css/enhanced.scss

```diff
@@ -47,7 +47,7 @@
 body {
   font-size: 18px;
   line-height: 24px;
-  max-width: 320px;
+  max-width: 400px;
 }
```

Comment:
@saeedmb8 for phones in landscape; ideally this should actually just resize to 100%, and take up all the available space on the device. But we're busy showing the site to people and they get confused when it takes up all the space.